style.css

```diff
@@ -72,12 +82,307 @@ body {
   background-image: url(img/header-shapes\ mobile.png);
   background-repeat: no-repeat;
   background-size: 100%;
-  left: -34px;
   top: 103.9993896484375px;
+  margin-bottom: 6rem;
 }
 
 .toggleNav {
   width: 18px;
   height: 12px;
   align-self: center;
 }
+
+/* Portfolio "Works" section */
+
+.works {
+  display: grid;
+  grid-template-rows: 4, 1fr;
+  row-gap: 5rem;
+}
+
+.portfolio2 {
+  box-sizing: border-box;
+  width: 100%;
+  height: auto;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  justify-self: center;
+}
+
+.mainContainer1 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.mainContainer2 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio2.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.mainContainer3 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio3.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
```

Comment:
- [ ] [OPTIIONAL] Kindly make your text aligned vertically on the same line as your `img` 🙏 
- [ ] [OPTIONAL] Kindly make sure you give design the same spacing as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205437082-664064b2-9057-43d0-8851-a15b26d56f1d.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205437091-3dbdd6c5-326a-4b82-b5ee-d9b2dcfe06b0.png)